docs/source/conf.py

```diff
@@ -0,0 +1,70 @@
+# type: ignore
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'anacreonlib'
+copyright = '2021, Ritik Mishra'
+author = 'Ritik Mishra'
+
+# The full version, including alpha/beta/rc tags
+release = '2.0.0a12'
```

Comment:
This release variable should be connected to whatever is in setup.py or something like that